src/common/main/kotlin/tech/thatgravyboat/skyblockapi/api/events/screen/InventoryChangeEvent.kt

```diff
@@ -16,7 +18,7 @@ class InventoryChangeEvent(
     val titleComponent: Component,
     val inventory: List<Slot>,
     val screen: AbstractContainerScreen<*>,
-) : SkyBlockEvent() {
+) : SkyBlockEvent(), ItemLike {
```

Comment:
idk how I feel about an event being itemlike feels a bit weird

Reply:
Does that mean i should remove it? Just kinda convenient for the tag keys
```kt
event in ItemTag.HUNTING_NOT_FOUND
```